packages/experiments/src/components/Button/Button.styles.ts

```diff
@@ -1,6 +1,7 @@
 import { IButtonComponent, IButtonStylesReturnType, IButtonTokenReturnType } from './Button.types';
 import { getFocusStyle, getGlobalClassNames, FontWeights, HighContrastSelector } from '../../Styling';
 import { IsFocusVisibleClassName } from '../../Utilities';
+import { parseGap } from 'office-ui-fabric-react/lib/components/Stack/StackUtils';
```

Comment:
Yeah, I was hoping to get your and @dzearing input here. Basically, I didn't want to rewrite the gap parsing function that lives inside `Stack` and the easiest way was to get it from this path, but I agree that it looks wrong in the context of a component styles file.

One thing we could do is make `Stack` export the `StackUtils` file which would allow us to use `parseGap` in other places just importing it from `office-ui-fabric-react` but it would increase the bundle-size of the `Stack` component.

Reply:
I think this is ok for now since this is an experimental component using an OUFR utility, but it should be changed once it's promoted to a relative path.